.github/workflows/Check_Tests.yml

```diff
@@ -0,0 +1,147 @@
+---
+name: CI-5974
+description: "Continuous Integration workflow for GHI 5974."
+run-name: Prototyping integration tests ${{ github.ref_name }}
+#
+# Jobs included:
+# - TEST-5974: Tests Integration with CPython Tests across Python versions and OSes
+#
+# Required Secrets:
+# NONE
+#
+# WORK IN PROGRESS
+# search for "TODO" in file for more details on what is still un-implemented
+
+
+on:  # yamllint disable-line rule:truthy
+  push:
+    branches: ["**"]  # matches any branch
+    tags: ["v*"]
```

Comment:
Would it make sense to specify `on: pull_request` too?

Reply:
@fanninpm I don't think so. as this isn't validating anything. only showing diagnostics about what we have left, it's not checking for potential regression (ATM).

In a way this is complementary to `https://github.com/RustPython/RustPython/blob/1aea1467da5269922feb6cdc099f2112b3ac90db/whats_left.py`

Reply:
@fanninpm

> Would it make sense to specify `on: pull_request` too?

:thinking: eventually to leverage the existing github annotations from these actions (and yes they already have those) on a per-file basis, it might make sense to add PRs, however it is too soon. So, I'm in agreement with @ShaharNaveh. These values probably should be even _more_ selective than they are to reduce noise of some branches.